health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/BoundaryPopup.js

```
@@ -0,0 +1,55 @@
+import { PopUp, Button } from "@egovernments/digit-ui-components";
+import { useTranslation } from "react-i18next";
+
+const BoundaryPopup = ({ showPopUp, setShowPopUp, callGeoPode, geoPodeData })=> {
+    const { t } = useTranslation();
+    return (
+        showPopUp &&  (
+            <PopUp 
+                className={"custom-popup-boundary"}
+                type={"default"}
+                heading={t("CHOOSE_MEANS_TO_CREATE_BOUNDARY")}
+                children={[
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)